File: tugas/php/Processor.php
<?php

class Processor{
    // atribut
    private $name, $price;

    // constructor
    public function __construct($name = "-", $price = 0){
        $this->name = $name;
        $this->price = $price;
    }

    // setter n getter
    public function setName($name){
        $this->name = $name;
    }

    public function getName(){
        return $this->name;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    public function getPrice(){
        return $this->price;
    }
}

File: tugas/java/Processor.java
public class Processor {
    // atribut
    private String name;
    private int price;

    // constructor
    public Processor(){}

    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    // setter n getter
    public void setName(String name) {
        this.name = name;
    }
    
    public String getName() {
        return name;
    }
    
    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return price;
    }
}